pkg/config/replica_config.go

```diff
@@ -105,7 +106,7 @@ func (c *ReplicaConfig) Marshal() (string, error) {
 	return string(cfg), nil
 }
 
-// UnmarshalJSON unmarshals into *ReplicationConfig from json marshal byte slice
```

Comment:
I guess this grammar is correct.

Reply:
yes, it's correct grammer, it was altered by IDE typo detector.